frontend/ui/test/services/rest_api_place_visit_service_test.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:tripmeout/model/place_visit.dart';
+import 'package:tripmeout/services/rest_api_place_visit_service.dart';
+
+void main() {
+  group('Populated RestApiPlaceVisitService', () {
+    var placeVisitService =
+        RestApiPlaceVisitService(endpoint: 'http://localhost:8080');
```

Comment:
Please only conditionally enable this test as in the [trip REST service test](https://github.com/googleinterns/step23-2020/blob/4223ce96c50ccae0d646f38e9e77c3153415bf3b/frontend/ui/test/services/rest_api_trip_service_test.dart#L8).